codejam-canvas/script/canvas.js

```diff
@@ -0,0 +1,45 @@
+const canvas = document.querySelector('#canvas');
+const ctx = canvas.getContext('2d');
+
+const img = document.createElement('img');
+img.src = './img/image.png';
+const choice = document.querySelector('.choice ul');
+
+
+function drawPic(frame, size, getColor) {
```

Comment:
another way is passing color as a string and boolean flag
```suggestion
function drawPic(frame, size, color, isRgba) {
```

Reply:
or even create separate function
`function getColor(color, isRgba) {...}`